package/python-kiwi-spec-template

```diff
@@ -95,57 +109,74 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 # python2-kiwi
 %package -n python2-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
+License:        GPL-3.0+
 Group:          Development/Languages/Python
-Provides:       python-kiwi = %{version}-%{release}
-Provides:       kiwi-image:tbz
+Provides:       kiwi
```

Comment:
Please don't do this, as this makes it difficult to have only one version installed when someone wants to install `kiwi`.

Also, I'm fairly certain that SUSE doesn't want to do this yet, so you may need to undo this change entirely.